File: .old/blazor-ipc-driven/backend/CoverMe.Backend.Core/Models/Process/ReportGeneratorOptions.cs
using CoverMe.Backend.Core.Enums.Process;

namespace CoverMe.Backend.Core.Models.Process;

public class ReportGeneratorOptions
{
    public required string ReportFilePath { get; init; }
    public required string OutputFolderPath { get; init; }
    public required ReportGeneratorReportType ReportType { get; init; } = ReportGeneratorReportType.Html;
}

File: src/src/rider/main/kotlin/com/jetbrains/rider/plugins/coverme/models/process/ReportGeneratorOptions.kt
package com.jetbrains.rider.plugins.coverme.models.process

import com.jetbrains.rider.plugins.coverme.enums.process.ReportGeneratorReportType

class ReportGeneratorOptions(
    val reportFilePath: String,
    val outputFolderPath: String,
    val reportType: ReportGeneratorReportType
)